### DISCUSSION TITLE
How to set the leaflet-popup-content width

### DISCUSSION THREAD

**Discussion by jozeflambrecht** [bozdoz/wp-plugin-leaflet-map#244] [Q&A]
Hello, I am testing this plugin, but was wondering how I can choose the width of the popup?
For example:
```
[leaflet-map lat=45.57083501292548 lng=-73.19165021181108 zoom=18 height="450"]

[leaflet-marker lat=45.571056574302446 lng=-73.19195866584779]

Ceci est un test.

[/leaflet-marker]

```
Gives me : 

![image](https://github.com/bozdoz/wp-plugin-leaflet-map/assets/11506957/d8a444eb-e312-40d7-ac54-ee7f1a10e2d4)

Seems like the width is a property of the element:
![image](https://github.com/bozdoz/wp-plugin-leaflet-map/assets/11506957/26c28ba9-42bc-4d72-ab23-5068f7eafa01)

**Comment by bozdoz** [comment]
Probably with custom css:

```css
.leaflet-popup-content {
  min-width: 100px;
}
```
